packages/dumb/src/components/stats/stat-legend/stat-legend.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2022, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { FunctionComponent, memo } from 'react'
+
+import colors from '../../../styles/colors.css'
+import styles from './stat-legend.css'
+import { Box } from '@material-ui/core'
+import { BgClasses } from '../models'
+import { StatLegendElement } from './stat-legend-element'
+
+interface StatLegendProps {
+  bgClasses: BgClasses
+  units: string
+}
+
+const StatLegend: FunctionComponent<StatLegendProps> = (props) => {
+  const { bgClasses, units } = props
+  const veryLowValue = Math.round(bgClasses.veryLow)
+  const lowValue = Math.round(bgClasses.low)
+  const targetValue = Math.round(bgClasses.target)
+  const highValue = Math.round(bgClasses.high)
+
+  return (
+    <Box data-testid="cbg-percentage-stats-legends" display="flex" marginLeft="8px" marginBottom="8px">
```

Comment:
The style of the `Box`es could be put in their dedicated CSS classes to avoid having custom style defined in the HTML

Reply:
It is common to define style in `Box` element in our code and we consider that ok. It is not the same as using the `style` props. At the end, a box is not using it as inline style. The `Box` component is made for this kind of thing (otherwise it would be unecessary as it would be the same as a `div`)